src/config/Config.ts

```diff
@@ -74,14 +74,15 @@ async function downloadConfig(
   configJsonFilename: string,
 ): Promise<ConfigOptions> {
   const url = new URL(configJsonFilename, window.location.href);
-  const res = await fetch(url);
+  const response = await fetch(url);
 
-  if (!res.ok || res.status === 404 || res.status === 0) {
+  // if we are running embedded on file:// origin then we can't trust the response status
+  if (!response.ok && (!response.url || !response.url.startsWith("file:"))) {
```

Comment:
Are you sure `res.status === 404 || res.status === 0` implies `response.ok === false` (just double checking to make sure the or condition was actually unnecassary.)

Reply:
why is it an `&&` condition if we cannot trust the response status?
It (`response.ok`) is still guaranteed to be false in a failure case but it is also false if it is a success?